libs/utils/utils.js

```diff
@@ -1,5 +1,9 @@
 /* eslint-disable no-console */
 
+import { enablePersonalizationV2, loadAnalyticsAndInteractionData, getEnv, getMetadata } from '../martech/helpers.js';
+
+export { getMetadata };
```

Comment:
Milo utils are *always* loaded for every page on adobe.com. Martech, is loaded under certain conditions at a certain time. This introduces a blocking network call for every single page on adobe.com:

![image](https://github.com/user-attachments/assets/b5253de1-af76-4abc-989f-b176332eac6e)

If we say the call really only takes 50ms as in this example (x10 this value for mobile with higher latency) and the current LCP is 500ms, this drops the performance by 10% - so there's a lot of gravity in introducing any blocking imports here.

Reply:
If we're willing to pollute the utils namespace with the code to make the interact call early, we maximize our performance benefits. We'll be trading organization for a performance improvement.